runHarness/Services/CassandraKubernetesService.pm

```diff
@@ -188,6 +188,26 @@ sub cleanData {
 	}	
 }
 
+sub waitForReady {
+	my ($self)   = @_;
+	my $logger = get_logger("Weathervane::Services::CassandraKubernetesService");
+	$logger->debug("waitForReady");
+	my $cluster = $self->host;
+	my $finished = 0;
+	while (!$finished) {
```

Comment:
Should there be a guard against infinite loop with some type of timeout failure?

Reply:
Yes.  I will add this, although for now I am going to make it a long timeout because the compaction may take a long time in the large configs.  I will add an issue to track adjusting the timeout later.